package.json

```diff
@@ -6,12 +6,15 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "update:changelog": "babel-node -- scripts/index.js"
+    "commit:changelog": "./scripts/commit_logs.sh",
+    "update:changelog": "babel-node -- scripts/index.js",
+    "deploy": "now --prod && now alias opencollective-changelog.now.sh changelog.opencollective.com",
+    "deploy:update": "npm run update:changelog && npm run deploy && npm run commit:changelog"
```

Comment:
All the commands are not following the spirit of other projects.

- `commit:changelog` -> `changelog:commit` (topic:action or maincommand:subcommand)
- `update:changelog` -> `changelog:update`
- `deploy:update` -> `update-and-deploy` 

Also, in Frontend, we use `npm-run-all` instead of doing `npm run command && npm run othercommand`, which is a best practice for Windows compatibility.